requirements.in

```diff
@@ -50,4 +50,5 @@ prance
 #-----------------------------------
 #   Other
 #-----------------------------------
-beautifulsoup4==4.12.2
+beautifulsoup4
+greenlet
```

Comment:
can you include the comment here please

---

arguably we then _only_ need it here and could remove it from data-store I think

Reply:
We need it, no? The reason we list it is because `pip-compile` is acting differently on different OS's.

Reply:
Yes. Can you include the comment explaining that here? We added one in data store

Reply:
Added